R/posterior_predict.R

```diff
@@ -819,6 +819,22 @@ posterior_predict_zero_inflated_binomial <- function(i, prep, ...) {
   ifelse(zi < theta, 0, rbinom(ndraws, size = trials, prob = prob))
 }
 
+posterior_predict_zero_inflated_beta_binomial <- function(i, prep, ...) {
+  # theta is the bernoulli zero-inflation parameter
+  theta <- get_dpar(prep, "zi", i = i)
+  trials <- prep$data$trials[i]
+  mu <- get_dpar(prep, "mu", i = i)
+  phi <- get_dpar(prep, "phi", i = i)
+  ndraws <- prep$ndraws
+  # beta location-scale probabilities
+  probs <- rbeta(ndraws, mu * phi, (1 - mu) * phi)
+  # compare with theta to incorporate the zero-inflation process
+  zi <- runif(ndraws, 0, 1)
+  ifelse(zi < theta, 
```

Comment:
could you make 1 line out of this if does not exceed 80 characters, or otherwise use the 
```
function_name(
  arg1 = ...
  arg2 = ...
  ...
)
```

style, for consistency?

Reply:
Yep sorry about that!